fonz/lookml.py

```diff
@@ -0,0 +1,122 @@
+from typing import List, Optional
+from fonz.exceptions import SqlError
+
+
+class LookMlObject:
+    def __repr__(self):
+        return f"{self.__class__.__name__}(name={self.name})"
+
+
+class Dimension(LookMlObject):
+    def __init__(self, name: str, type: str, sql: str, url: str):
+        self.name = name
+        self.type = type
+        self.sql = sql
+        self.url = url
+        self.ignore = True if "fonz: ignore" in sql else False
```

Comment:
Is it currently implemented to actually ignore based on this?

Reply:
Not yet!